test/cli_lang_boilerplate_test.go

```diff
@@ -29,6 +29,8 @@ var Runtimes = []struct {
 }{
 	{"go", ""},
 	{"dotnet", ""},
+	{"dotnet6.0", ""},
```

Comment:
can you reverse this order to be inline with other's

Reply:
Yeah done, thanks!